plugins/python-build/bin/python-build

```diff
@@ -876,6 +876,25 @@ build_package_pyston() {
   shopt -u nullglob
 }
 
+build_package_pyston() {
+  # currently supported version 2.3.1 and higher
+  build_package_copy
+  mv "${PREFIX_PATH}/usr/bin" "${PREFIX_PATH}/bin"
+  mv "${PREFIX_PATH}/usr/lib" "${PREFIX_PATH}/lib"
+  local bin
+  shopt -s nullglob
+  for bin in "usr/bin/"*; do
```

Comment:
Not sure... won't this always be a no-op since you've just moved `usr/bin` away? What's the CWD here, and how do you know you can rely on it?

Reply:
This code also looks overcomplicated. If all you need is to symlink `pythonX.Y-*` to `python`, you can just take the first result of `${PREFIX_PATH}/bin/python?.?-*`

Reply:
```
ln -s "$(basename "${PREFIX_PATH}/bin/python?.?-*")" "${PREFIX_PATH}/bin/python"
```

Reply:
Sounds good. Looks like the code might not be necessary at all; I tested to make sure that a `python` alias was being written, but clearly it's not being written by my non-functional code here, so maybe there's something else that's writing it? My guess is that the code I copied this from was only necessary in order to deal with the fact that we didn't used to have anything called `python*`, so maybe we can just remove it?

Reply:
It's your job to test if a code change works :-)

Reply:
I can be snarky too and say that I followed all of the steps that were provided :)

Reply:
Looks like there was a misunderstanding. I wrote that solely because you seemed to be requesting me to test your changes for you. If after removing the code, the installer still works as intended then of course it's okay to remove it...